doc/dev/dev_cluster_deployement.rst

```diff
@@ -73,6 +73,23 @@ Options
 
     Launch the osd/mds/mon/all the ceph binaries using valgrind with the specified tool and arguments.
 
+.. option:: --[{mon,osd,mds}]_num
+
+    Set the count of monitors, osds and mds
```

Comment:
better off putting 

> Set the count of monitors, osds or mds

? i am referencing the notation of "osd/mds/mon/all" above, where "/" always means "or", i guess.